### DISCUSSION TITLE
[PROPOSAL] Splitting Boost Collectibles

### DISCUSSION THREAD

**Discussion by brian-man** [sunflower-land/sunflower-land#2332] [General]
Credit to `GodIsGreat` for the original concept.

### Overview
Some of the boost collectibles are very strong.  So strong that it might be interesting to open up a feature to split them.

- Distributing to smaller pieces will lead to less impact on game economy because of boost distribution.
- When someone buys fully boosted collectible, they are probably going to maximize profits but not as much when distributed.
 
### Example
Iron Idol increases iron drops by +1.  That's incredibly strong given the lack of iron boosts in the game currently.  So strong that some owners feel a bit uncomfortable about how strong it is.  Wouldn't it be nice if it could be split into 10 Iron Idol Fragments and sell them individually?

### Strawman Implementation Draft
1. Allow deposit and withdraw of 0.1 of splittable collectibles (as well as the normal 1)
2. Players can freely transfer/buy/sell 0.1 of splittable collectibles on the open market
3. With 2, players can naturally "combine" 10 of 0.1 supply into 1 supply or the reverse
4. In game, allow placing and removal of 0.1 of splittable collectibles
5. In game, when placing 0.1 of a splittable collectable the name (in tooltips and elsewhere) used could be either the original name ("Iron Idol") or a specialized name ("Iron Idol Fragment")
6. When delivering land data to the client, the json should report the size (1 for full, 0.1 for fragment)
7. Game front end should apply 10% of the boost when "only" a fragment is placed

Note that none of this proposal requires updating the SLI contract or other changes to the blockchain.

**Comment by Life-Is-Great** [comment]
I fully support this one.

Your part made it so unique m8.
Changing decimals for this nft would be so easy implementation and so outside box one.

Not sure if it would be possible but could ad to this:
1) Changing decimals on blockchain
2) When user views it after this change it would show 10x of item
3) If he owns 10x then he gets full boost
4) If he owns 1 then he gets full boost / 10 

?

**Reply by brian-man** [reply]
> Not sure if it would be possible but could ad to this:
> 1. Changing decimals on blockchain
> 2. When user views it after this change it would show 10x of item
> 3. If he owns 10x then he gets full boost
> 4. If he owns 1 then he gets full boost / 10

That could be done but would be more work and impact blockchain state.  The latter of course meaning it trigger OpenSea to go insane again.  It also would maybe limit the feature only to collectibles first minted after the change if the team has other reasons why fiddling with decimals is bad.

**Reply by Life-Is-Great** [reply]
Sure.
Aware of that. OS did have those decimal issues before yes.
And if this is to be implemented in this way then it is something to keep in mind.

Alternative is to burn old item and give 10x of lower tier ones or create new item that is 10% of the original boost

**Comment by NotKeen** [comment]
I feel like splitting up might be too confusing / not intuitive. But the current release of cabbage girl and boy go in a similar direction, where they get a bonus if you have both. I think this kind of tier (or set item) system / SFTs building on each other is interesting and should be explored more in future releases.
